lib/pages/homepage.dart

```diff
@@ -1,19 +1,117 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_slidable/flutter_slidable.dart';
+import 'package:timetable_2_flutter_admin/globals/mySpaces.dart';
 import 'package:timetable_2_flutter_admin/globals/sizeConfig.dart';
 import 'package:timetable_2_flutter_admin/widgets/appBar.dart';
+import 'package:timetable_2_flutter_admin/widgets/myListTile.dart';
 
-class HomePage extends StatelessWidget {
+class HomePage extends StatefulWidget {
+  @override
+  _HomePageState createState() => _HomePageState();
+}
+
+class _HomePageState extends State<HomePage> {
   @override
   Widget build(BuildContext context) {
     SizeConfig().init(context);
-    print(SizeConfig.screenHeight);
-    print('Vertical Blocks = ${SizeConfig.verticalBlockSize}');
-    print(40 / SizeConfig.verticalBlockSize);
     return Scaffold(
       appBar: appBar(),
-      body: Center(
-        child: Text('Approval Page'),
+      body: Padding(
+        padding: const EdgeInsets.fromLTRB(10, 0, 10, 0),
```

Comment:
Prefer writing EdgeInsets.symmetric(horizontal:10) in such a case. It does the same thing. Practically for all purposes.. EdgeInsets.all() and EdgeInsets.only() and EdgeInsets.symmetric() suffice.

Reply:
Ok! I'll update in the next commit